conf/testfm.sample.yaml

```diff
@@ -12,5 +12,13 @@
   # DOGFOOD_ORG: <DOGFOOD_ORG>
   # DOGFOOD_ACTIVATIONKEY: <DOGFOOD_ACTIVATIONKEY>
   # CAPSULE_DOGFOOD_ACTIVATIONKEY: <CAPSULE_DOGFOOD_ACTIVATIONKEY>
-# TESTFM:
-  # HOTFIX_URL: <HOTFIX_URL>
+
+# copy ROBOTTELO section from robottelo.yaml or keep robottelo.yaml under conf/
+# ROBOTTELO:
+  # - The URL of container hosting repos on SatLab
+  # REPOS_HOSTING_URL: <repo_hosting_url>
+  # Satellite version supported by this branch
+  # Set this via envvar when building robottelo-container for z-stream branches
+  # SATELLITE_VERSION: '7.0'
+  # The Base OS RHEL Version(x.y) where the satellite would be installed
+  # RHEL_VERSION: '7.9'
```

Comment:
Do we use these settings in testfm? If not, then do we need to mention it?

Reply:
added this here just to show a few required settings, like sat and rhel versions which might be required for some tests